packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/FormRow/FormSections/SegmentCollectionItems/CuratorialAssessmentItem.js

```diff
@@ -79,8 +80,7 @@ class CuratorialAssessmentItem extends PureComponent {
     return (
       <List.Item>
         <List.Content style={{ padding: '0.5em' }} verticalAlign="bottom">
-          <Modal
-            className="ui form"
+          <FormModal
```

Comment:
it doesn't look like it's needed in this case, because the curatorial assessment form is using the semantic Form component